tools/mason/MasonHelp.chpl

```diff
@@ -187,6 +186,8 @@ proc masonSystemSearchHelp() {
   writeln();
   writeln("Options:");
   writeln("    -h, --help                  Display this message");
+  writeln("        --no-desc               Only display package name");
+  writeln("        --hungry                Parse descriptions of package to include more search results");
```

Comment:
Interesting flag name...

Reply:
@ben-albrecht  It's hungry! it consumes more for the search! It made sense to me...other suggestions?

Reply:
Some potentially more intuitive flag names (at the expense of fun), ordered roughly by my preference:

- `--search-desc`
- `--extended`
- `--broad`
- `--advanced`

Reply:
How about ``--[no-]show-desc`` to toggle display of descriptions, and ``--[no-]desc`` to toggle searching of descriptions? FWIW, homebrew uses ``--desc`` as well.

Reply:
> FWIW, homebrew uses --desc as well.

Ah, I did not notice this. Good point. Those would work for me.